node-basic/start_actor.js

```diff
@@ -0,0 +1,46 @@
+const { execSync } = require('child_process');
+const path = require('path');
+
+/**
+ * This is the start function for Apify Actor. It runs node script from packageJson.scripts.start.
+ * If there is no start script it will use "node main.js" as a fallback. It is for backward compatibility with older actors.
+ * If there is start script, but it is not a node command it executes the command in a child process.
+ */
+const startActor = () => {
+    let packageJson;
+    try {
+        packageJson = require('./package.json');
+    } catch (err) {
+        // There is no package.json
+    }
+
+    let startScript;
+    if (packageJson && packageJson.scripts && packageJson.scripts.start) {
+        startScript = packageJson.scripts.start;
+    } else {
+        // If we don't find start script, use fallback to old "node main.js"
+        console.log('\nWARNING: The npm start script not found in package.json. Using "node main.js" instead. Please update your package.json file. For more information see https://github.com/apifytech/apify-cli/blob/master/MIGRATIONS.md\n\n');
+        startScript = 'node main.js'
+    }
+
+    // Check if start script includes node command with javascript file
+    const match = startScript.match(/node\s+(.*\.js)/);
```

Comment:
Why only command ending with js file? What if the command has some arguments, then this will not work, but why shouldn't it? Maybe just check if the command starts with `"node "` would be enough? Anyway, some explanation in comments would be great

Reply:
You are right, this will not work with env vars or flags in the start script... We have to find another way.

Reply:
I'd just check if the command starts with "node ", that should be good enough

Reply:
Oh, I see you removed the start script completely ... I guess it might be fine, if we know there are no more people using the legacy setting, as we should know based on the new logging soon...